index.html

```diff
@@ -289,6 +289,33 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+<section class="contact-me">
+  <div>
+  <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+  </div>
+  <div>
+  <form action="https://formspree.io" method="post">
+  <input type="text" id="name" name="name" placeholder="Full Name" maxlength="90" required>
+  <input type="text" id="email" name="email" placeholder="Email Addres" required>
```

Comment:
- [ ]  Kindly check for the  type value for the email, as you are  using this input to receive an email the type sould be set as ` type = 'email'`